<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>317</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Global Hotkey Manager</string>
  </property>
  <widget class="QLabel" name="lblListTitle">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>112</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Global hotkeys (All 0)</string>
   </property>
   <property name="buddy">
    <cstring>lstHotkeys</cstring>
   </property>
  </widget>
  <widget class="QListWidget" name="lstHotkeys">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>27</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>256</y>
     <width>366</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblHint">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>265</y>
     <width>310</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: You can press Ctrl+Shift+Alt+F12 to open this window.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Detail</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>&amp;Add...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRemove">
      <property name="text">
       <string>&amp;Remove</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editBtn">
      <property name="text">
       <string>&amp;Edit...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnInvoke">
      <property name="text">
       <string>&amp;Invoke</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnRefresh">
      <property name="text">
       <string>Re&amp;fresh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHide">
      <property name="text">
       <string>&amp;Hide</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>E&amp;xit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lstHotkeys</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>editBtn</tabstop>
  <tabstop>btnInvoke</tabstop>
  <tabstop>btnRefresh</tabstop>
  <tabstop>btnHide</tabstop>
  <tabstop>btnExit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
